### PR TITLE
LPS 185980 Cannot set file size limits for image/svg+xml mimeType

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Review by marcogalluzzi — APPROVED** [review]

**Comment by marcogalluzzi** [comment]
ci:forward

**Comment by marcogalluzzi** [comment]
Hi @robertoDiaz I've noticed that mime types could have these characters: ".", ";", "=". Examples:
* `application/vnd.ms-spreadsheetml`
* `application/x-x509-cert;format=pem`

Wouldn't be a good idea to also add these characters?

Source: [Apache Tika](https://github.com/apache/tika/blob/main/tika-core/src/main/resources/org/apache/tika/mime/tika-mimetypes.xml)

**Comment by robertoDiaz** [comment]
I'll do so

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/MimeTypeSizeLimitUtil.java [hunk 1]
```diff
@@ -96,6 +96,6 @@ private static Long _parseSizeLimit(String sizeLimit) {
 		MimeTypeSizeLimitUtil.class);
 
 	private static final Pattern _pattern = Pattern.compile(
-		"[a-zA-Z0-9][a-zA-Z0-9$!#&^_+-]*");
+		"[a-zA-Z0-9][a-zA-Z0-9$!#&.,;=^+-]*");
```
Comment (marcogalluzzi) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/MimeTypeSizeLimitUtil.java:99]: Why did you remove the underscore character?

**Review by marcogalluzzi — CHANGES_REQUESTED** [review]

**Review by marcogalluzzi — APPROVED** [review]

**Comment by marcogalluzzi** [comment]
ci:forward
